omero/sysadmins/customization.txt

```diff
@@ -48,24 +48,32 @@ Top links menu
 
 :property:`omero.web.ui.top_links` adds links to the top header::
 
-    $ bin/omero config append omero.web.ui.top_links '["Figure", "webfigure"]'
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index", {"title": "Open Figure in new tab", "target": "_blank"}]'
     $ bin/omero config set omero.web.ui.top_links '["GRE", "http://lifesci.dundee.ac.uk/gre"]'
```

Comment:
I noticed we've got a ```set``` here, which will wipe out all the top links and replace them with this single link. This should be updated to use ```append``` as above. I'm sure that we mention the ```set``` option elsewhere without having to give an example of it here.